### ISSUE TITLE
Provide the number of k-mers in the query

### ISSUE THREAD

**Issue by graceblackwell** [bingmann/cobs#13]
Could the number of k-mers in the query be included in the output so the user can calculate the percentage of the query in each dataset (or provide the percentage as well)?

**Comment by bingmann** [comment]
Maybe we should define a better output format.

**Comment by graceblackwell** [comment]
Yeah could be useful! Would be nice to have it write to a file as well. Could do something like bigsi? Though have the query header instead of the query sequence

**Comment by iqbal-lab** [comment]
Output format an excellent idea.
